src/main/resources/SQL_Scripts/database_creation_script.sql

```diff
@@ -1,120 +1,134 @@
-
 CREATE TABLE public.user_Account (
-                email VARCHAR(254) NOT NULL,
-                first_name VARCHAR(100) NOT NULL,
-                last_name VARCHAR(100) NOT NULL,
-                password VARCHAR(100) NOT NULL,
-                balance NUMERIC(6,2) NOT NULL,
-                is_activated BOOLEAN DEFAULT true NOT NULL,
-                CONSTRAINT user_account_pk PRIMARY KEY (email)
+    email        VARCHAR(254)         NOT NULL,
+    first_name   VARCHAR(100)         NOT NULL,
+    last_name    VARCHAR(100)         NOT NULL,
+    password     VARCHAR(100)         NOT NULL,
+    balance      NUMERIC(6, 2)        NOT NULL,
+    is_activated BOOLEAN DEFAULT true NOT NULL,
+    CONSTRAINT user_account_pk PRIMARY KEY (email)
 );
 
+CREATE TABLE public.authorities (
+    email     VARCHAR(254) NOT NULL,
+    authority VARCHAR(50)  NOT NULL,
+    CONSTRAINT authorities_pk PRIMARY KEY (email)
+);
 
 CREATE SEQUENCE public.bank_account_id_seq;
 
 CREATE TABLE public.bank_account (
-                id INTEGER NOT NULL DEFAULT nextval('public.bank_account_id_seq'),
-                holder_first_name VARCHAR(100) NOT NULL,
-                holder_last_name VARCHAR(100) NOT NULL,
-                iban VARCHAR(31) NOT NULL,
-                bic VARCHAR(11) NOT NULL,
-                user_account_email VARCHAR(254) NOT NULL,
-                CONSTRAINT bank_account_pk PRIMARY KEY (id)
+    id                 INTEGER      NOT NULL DEFAULT nextval('public.bank_account_id_seq'),
+    holder_first_name  VARCHAR(100) NOT NULL,
```

Comment:
Does 100 characters being enough? I mean, in this case yes surely, but aren't we used to put sth more like 255?

Reply:
we can discuss it tomorrow so :)